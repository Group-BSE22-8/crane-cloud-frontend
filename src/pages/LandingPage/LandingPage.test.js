/* eslint-disable no-undef */
import React from "react";

import { shallow } from "enzyme";

import LandingPage, { mapStateToProps } from "./";

const LandingPageProps = {
  user: {
    data: {
      id: undefined,
    },
  },
};
const userDefinedProps = {
  user: {
    data: {
      id: "test",
    },
  },
};
describe("LandingPage component test", () => {
  let landingpageWrapper, landingpageUserWrapper;
  beforeEach(() => {
    const LandingPagewrapper = LandingPage.WrappedComponent;
    landingpageWrapper = shallow(<LandingPagewrapper {...LandingPageProps} />);
    landingpageUserWrapper = shallow(<LandingPagewrapper {...userDefinedProps} />);
  });
  it("matches rendered component with the existing snapshot", () => {
    expect(landingpageWrapper).toMatchSnapshot();
  });
});
describe("Test the landing page map state to props", () => {
  it("checks the result of map state to props function", () => {
    expect(
      mapStateToProps({
        user: {},
      })
    ).toEqual({
        user: {},
    });
  });
});
