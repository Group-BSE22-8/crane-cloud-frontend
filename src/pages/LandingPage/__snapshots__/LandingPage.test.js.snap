// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component test matches rendered component with the existing snapshot 1`] = `ShallowWrapper {}`;
